src/drafts/SelectPanel2/SelectPanel.stories.tsx

```diff
@@ -428,29 +421,32 @@ export const TODO1Uncontrolled = () => {
 
 export const TODO2SingleSelection = () => <h1>TODO</h1>
 
-export const TODO3NoCustomisation = () => {
-  return (
-    <>
-      <h1>TODO: Without any customisation</h1>
-      <p>Address after TODO: Uncontrolled</p>
-    </>
-  )
-}
-
-export const TODO4WithFilterButtons = () => {
+export const FWithFilterButtons = () => {
```

Comment:
Your story names remind me of a PR that I opened a while ago and abandoned 😬 https://github.com/primer/react/pull/3588 I'll look into it 👀

Reply:
🤭 thank you! it's such a silly hack